blocks/trending-news/trending-news.css

```diff
@@ -0,0 +1,173 @@
+    line-height: 1.5rem;
+    font-weight: var(--font-weight-bold);
+    color: var(--link-color-orange);
+    text-decoration: none;
+}
+
+.block.trending-news .container .title {
+    margin-bottom: 2rem;
+    text-align: center;
+}
+
+.block.trending-news .container .title h2 {
+    font-size: var(--heading-font-size-xmxl);
+    color: var(--primary);
+    line-height: 2.813rem;
+    text-transform: uppercase;
+    font-weight: var(--font-weight-bold);
+}
+
+.block.trending-news .news-slider .news-track {
+    padding-left: 1rem;
+    position: relative;
+    left: 0;
+    top: 0;
+    display: flex;
+    transition: transform 0.5s ease;
+    min-height: 17.5rem;
+    transform: translate3d(0, 0, 0);
+    overflow: hidden;
+}
+
+.block.trending-news .news-slider .news-card {
+    width: 25rem;
+    height: 20rem;
+    flex: 0 0 auto;
+    min-height: var(--space-1px);
+    transition: opacity 0.1s ease;
+}
+
+.block.trending-news .news-section .article {
+    background: transparent;
+    position: relative;
+    border: var(--space-1px) solid var(--boarder-trendign-news-card);
+    height: 100%;
+}
+
+.block.trending-news .news-section .article .picture-wrapper img {
+    object-fit: initial;
+    width: 23.375rem;
+    height: 10rem;
+    display: block;
+    transition: 0.3s;
+}
+
+.block.trending-news .news-section .article .article-text {
+    padding: var(--space-10px) 0;
+}
+
+.block.trending-news .news-section .article .article-text a {
+    padding-right: 2rem;
+    margin-bottom: 1.125rem;
+    font-size: var(--space-15px);
+    line-height: 1.375rem;
+    color: var(--color-black);
+    height: 2.625rem;
+    font-weight: var(--font-weight-medium);
+    word-break: break-word;
+    overflow: hidden;
+    text-overflow: ellipsis;
+    display: -webkit-box;
+    -webkit-line-clamp: 2;
+    -webkit-box-orient: vertical;
+    text-decoration: none;
+}
+
+.block.trending-news .news-section .article .article-text .post-info {
+    display: flex;
+    flex-wrap: wrap;
+    margin-top: var(--space-5px);
+}
+
+.block.trending-news .news-section .article .article-text .post-info img {
+    width: var(--space-12px);
+    height: var(--space-12px);
+    margin-right: var(--space-5px);
+    display: block;
+}
+
+.block.trending-news .news-section .article .article-text .post-info abbr {
+    font-size: var(--space-11px);
+    color: var(--card-post-color);
+    padding-right: var(--space-5px);
+    border-right: var(--space-1px) solid var(--card-post-boarder);
+    line-height: var(--space-10px);
+    margin: 0 var(--space-4px) var(--space-8px) 0;
+}
+
+.block.trending-news .discover-more.text-right {
+    margin-top: 1rem;
+    text-align: center;
+}
+
+.block.trending-news .discover-more .icon {
+    margin-left: 0.5rem;
+    font-family: icomoon, sans-serif;
+    font-style: normal;
+    font-weight: var(--font-weight-medium);
+    font-variant: normal;
+    text-transform: none;
+    line-height: 1;
+    -webkit-font-smoothing: antialiased;
```

Comment:
@ravverma just confirm if we need this ?

Reply:
nope removed